src/js/media-controller.js

```diff
@@ -775,6 +792,12 @@ class MediaController extends MediaContainer {
     switch (e.key) {
       case ' ':
       case 'k':
+        if (!this.#hotkeyEnabled('Space') &&
```

Comment:
suggestion(non-blocking): Should we just check:

```js
if (!this.#hotkeyEnabled(e.key) return;
```
once up top?

Reply:
yes, would be neat